December/09/grace_hopper/info.yaml

```diff
@@ -0,0 +1,13 @@
+title: Grace Hopper
+description: |
+    On December 09, 1906 was born Grace Brewster Murray Hopper.
+    She is an American computer scientist and United States Navy rear admiral.
+    Hopper earned a Ph.D. (1934) in mathematics from Yale and she best known for her trailblazing contributions to computer programming, software development, 
+    and the design and implementation of programming languages.
+    Hopper is credited with popularizing the term "debugging" and "bug", as during her team work on Harvard Mark II computer, her associates discovered a moth that was stuck in a relay 
+    and was impeding the operation of the computer. Upon extraction, the insect was affixed to a log sheet for that day with the notation, "First actual case of a bug being found".
+url: https://en.wikipedia.org/wiki/Grace_Hopper
+images:
+    - filename: grace_hopper.jpg
```

Comment:
The file itself is 600x600 px but has been distorted and the proportions/aspect ratio don't match the original. 

Take the closest image by size and crop the edges, keeping the proportions instead of resizing disproportionately.